tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -8,13 +8,20 @@
 """
 import pytest
 
-from ..models.utils import check_outputs_equal
+from ..models.utils import check_logprobs_close, check_outputs_equal
 
 MODELS = [
     "facebook/opt-125m",
     "meta-llama/Llama-2-7b-hf",
 ]
 
+FP8_MODELS = [
+    # does not work - CUDA illegal memory access - undiagnosed
+    # "facebook/opt-125m",
```

Comment:
We should not allow known potential illegal memory access issue. At least we need to know what workloads/models would likely trigger that, and have an issue for people to report and track progress.

Reply:
Should I file an issue for it?

Well landing this is still better than error for every model, but you're right we should track it.

Reply:
Yes please file an issue. We don't want users to randomly see this error but cannot find any related resource.

Reply:
Created issue: https://github.com/vllm-project/vllm/issues/7378